spring-cloud-gcp-data-datastore/src/test/java/org/springframework/cloud/gcp/data/datastore/it/DatastoreIntegrationTests.java

```diff
@@ -209,7 +209,7 @@ public void testFindByExample() {
 		assertThat(this.testEntityRepository
 				.findAll(
 						Example.of(new TestEntity(null, null, null, null, null)),
-						Sort.by(Sort.Direction.ASC, "size")))
```

Comment:
Was this test supposed to test sorting by size? Perhaps sort by both, "size" and "id" to only work as tie-breaker? @dmitry-s WDYT?

Reply:
I think it looks fine, sort by `id` makes sense